js/controller.js

```diff
@@ -1362,9 +1362,22 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
     showVolumeSliderBar: function() {
       this.state.volumeState.volumeSliderVisible = true;
+      if (Utils.isAndroid()) {
+        this.startHideVolumeSliderTimer();
+      }
       this.renderSkin();
     },
 
+    startHideVolumeSliderTimer: function() {
+        this.cancelTimer();
+        var timer = setTimeout(function() {
```

Comment:
I was worried about unexpected state transitions resulting in timers being left around and not canceled.  But now I see that it's handled.  So you can disregard my concern.